### ISSUE TITLE
Stack Trace is Illegible 

### ISSUE THREAD

**Issue by sherl0cks** [Microsoft/vscode-java-test#226]
This is the text in the test output window. I would like to see the new lines properly formatted so the stack trace is legible:

```
@@<{"name":"testReporterAttached"}>
@@<{"name":"rootName", "attributes":{"name":"Default Suite"}}>
@@<{"name":"suiteTreeStarted", "attributes":{"name":"io.openinnovationlabs.KafkaClientTest", "location":"java:test://io.openinnovationlabs.KafkaClientTest"}}>
@@<{"name":"suiteTreeNode", "attributes":{"name":"shouldSendARecordToKafka", "location":"java:test://io.openinnovationlabs.KafkaClientTest.shouldSendARecordToKafka"}}>
@@<{"name":"suiteTreeEnded", "attributes":{"name":"io.openinnovationlabs.KafkaClientTest", "location":"java:test://io.openinnovationlabs.KafkaClientTest"}}>
2018-07-03 18:48:04 INFO  KafkaAdapter:24 - initialized
2018-07-03 18:48:05 INFO  Log4jControllerRegistration$:31 - Registered kafka:type=kafka.Log4jController MBean
@@<{"name":"testFailed", "attributes":{"name":"null", "message":"org/apache/kafka/common/record/RecordFormat", "details":"java.lang.NoClassDefFoundError: org/apache/kafka/common/record/RecordFormat\n\tat kafka.api.KAFKA_0_8_0$.<init>(ApiVersion.scala:121)\n\tat kafka.api.KAFKA_0_8_0$.<clinit>(ApiVersion.scala)\n\tat kafka.api.ApiVersion$.<init>(ApiVersion.scala:46)\n\tat kafka.api.ApiVersion$.<clinit>(ApiVersion.scala)\n\tat kafka.server.Defaults$.<init>(KafkaConfig.scala:142)\n\tat kafka.server.Defaults$.<clinit>(KafkaConfig.scala)\n\tat kafka.server.KafkaConfig$.<init>(KafkaConfig.scala:752)\n\tat kafka.server.KafkaConfig$.<clinit>(KafkaConfig.scala)\n\tat kafka.server.KafkaConfig.NumPartitionsProp(KafkaConfig.scala)\n\tat io.debezium.kafka.KafkaServer.populateDefaultConfiguration(KafkaServer.java:95)\n\tat io.debezium.kafka.KafkaServer.<init>(KafkaServer.java:77)\n\tat io.debezium.kafka.KafkaServer.<init>(KafkaServer.java:61)\n\tat io.debezium.kafka.KafkaCluster.lambda$addBrokers$0(KafkaCluster.java:126)\n\tat java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1660)\n\tat io.debezium.kafka.KafkaCluster.addBrokers(KafkaCluster.java:124)\n\tat io.openinnovationlabs.KafkaClientTest.spinUpKafka(KafkaClientTest.java:50)\n\tat io.openinnovationlabs.KafkaClientTest.init(KafkaClientTest.java:34)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat io.vertx.ext.unit.junit.VertxUnitRunner.invokeTestMethod(VertxUnitRunner.java:95)\n\tat io.vertx.ext.unit.junit.VertxUnitRunner.lambda$invokeExplosively$0(VertxUnitRunner.java:114)\n\tat io.vertx.ext.unit.impl.TestContextImpl$Step.run(TestContextImpl.java:128)\n\tat io.vertx.ext.unit.impl.TestContextImpl$Step.access$200(TestContextImpl.java:30)\n\tat io.vertx.ext.unit.impl.TestContextImpl.run(TestContextImpl.java:237)\n\tat io.vertx.ext.unit.junit.VertxUnitRunner.invokeExplosively(VertxUnitRunner.java:130)\n\tat io.vertx.ext.unit.junit.VertxUnitRunner.access$000(VertxUnitRunner.java:39)\n\tat io.vertx.ext.unit.junit.VertxUnitRunner$2.evaluate(VertxUnitRunner.java:194)\n\tat io.vertx.ext.unit.junit.VertxUnitRunner$3.evaluate(VertxUnitRunner.java:211)\n\tat org.junit.runners.ParentRunner.run(ParentRunner.java:363)\n\tat com.microsoft.java.test.runner.JUnit4TestReference.run(JUnit4TestReference.java:51)\n\tat com.microsoft.java.test.runner.CustomizedJUnitCoreRunner.run(CustomizedJUnitCoreRunner.java:45)\n\tat com.microsoft.java.test.runner.JUnitLauncher.main(JUnitLauncher.java:21)\nCaused by: java.lang.ClassNotFoundException: org.apache.kafka.common.record.RecordFormat\n\tat java.net.URLClassLoader.findClass(URLClassLoader.java:381)\n\tat java.lang.ClassLoader.loadClass(ClassLoader.java:424)\n\tat sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)\n\tat java.lang.ClassLoader.loadClass(ClassLoader.java:357)\n\t... 34 more\n", "duration":"1530665286021"}}>
@@<{"name":"testFailed", "attributes":{"name":"null", "message":"null", "details":"java.lang.NullPointerException\n\tat io.openinnovationlabs.KafkaClientTest.tearDown(KafkaClientTest.java:40)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat io.vertx.ext.unit.junit.VertxUnitRunner.invokeTestMethod(VertxUnitRunner.java:95)\n\tat io.vertx.ext.unit.junit.VertxUnitRunner.lambda$invokeExplosively$0(VertxUnitRunner.java:114)\n\tat io.vertx.ext.unit.impl.TestContextImpl$Step.run(TestContextImpl.java:128)\n\tat io.vertx.ext.unit.impl.TestContextImpl$Step.access$200(TestContextImpl.java:30)\n\tat io.vertx.ext.unit.impl.TestContextImpl.run(TestContextImpl.java:237)\n\tat io.vertx.ext.unit.junit.VertxUnitRunner.invokeExplosively(VertxUnitRunner.java:130)\n\tat io.vertx.ext.unit.junit.VertxUnitRunner.access$000(VertxUnitRunner.java:39)\n\tat io.vertx.ext.unit.junit.VertxUnitRunner$3.evaluate(VertxUnitRunner.java:217)\n\tat org.junit.runners.ParentRunner.run(ParentRunner.java:363)\n\tat com.microsoft.java.test.runner.JUnit4TestReference.run(JUnit4TestReference.java:51)\n\tat com.microsoft.java.test.runner.CustomizedJUnitCoreRunner.run(CustomizedJUnitCoreRunner.java:45)\n\tat com.microsoft.java.test.runner.JUnitLauncher.main(JUnitLauncher.java:21)\n", "duration":"1530665286023"}}>
Total tests run: 0, Failures: 2, Skips: 0
``

**Comment by ansyral** [comment]
@sherl0cks You can view formatted stack trace in test report. Click on test result on the status bar or codelens can open the test report.

**Comment by sherl0cks** [comment]
My stack trace is happening in a @BeforeClass, there does not appear to be a way to view this in the test report.

**Comment by ansyral** [comment]
@sherl0cks I got your idea now.

**Comment by jdneo** [comment]
Should be fixed in #556
